frontend/src/components/NavBar.js

```diff
@@ -44,6 +53,15 @@ class NavBar extends Component {
     const { isPM } = this.props
     return (
       <div>
+        <Sidebar
+          rootClassName="sidebar-root"
+          sidebar={<NotificationsBar />}
+          open={this.state.sidebarOpen}
+          onSetOpen={this.onSetSidebarOpen}
+          pullRight={true}
+          styles={{ sidebar: { background: 'white' } }}
```

Comment:
I'm thinking that this can moved into the css file or it might not be needed unless the sidebar isn't already defaulted to a white background